test/test_block_load_store.cu

```diff
@@ -517,7 +517,7 @@ int main(int argc, char** argv)
 
     // Compile/run thorough tests
 #if TEST_VALUE_TYPES == 0
-    TestThreads<char>(2, 0.8f);
```

Comment:
I am slightly surprised that we only test signed integers but that is definitely preexisting, so fine by me

Reply:
For load facility, I don't think it's reasonable to test multiple types of the same size.